api/swig/python/setup.py.in

```diff
@@ -0,0 +1,35 @@
+import setuptools.command.install
+import shutil
+from distutils.sysconfig import get_python_lib
+
+
+class CompiledLibInstall(setuptools.command.install.install):
+    """
+    Specialized install to install to python libs
+    """
+
+    def run(self):
+        """
+        Run method called by setup
+        :return:
+        """
+        # Get filenames from CMake variable
+        filenames = '${PYTHON_INSTALL_FILES}'.split(';')
+
+        # Directory to install to
+        install_dir = get_python_lib()
+
+        # Install files
+        [shutil.copy(filename, install_dir) for filename in filenames]
```

Comment:
This installation assumes that all the shared libraries required by baglib.dll are in the environment. It is often not the case.

Reply:
Will address in version 2.0.2, see: https://github.com/OpenNavigationSurface/BAG/issues/69.